payouts_test.go

```diff
@@ -0,0 +1,196 @@
+package goshopify
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/jarcoal/httpmock"
+)
+
+func TestPayoutList(t *testing.T) {
+	setup()
+	defer teardown()
+
+	httpmock.RegisterResponder("GET", fmt.Sprintf("https://fooshop.myshopify.com/%s/shopify_payments/payouts.json", client.pathPrefix),
+		httpmock.NewStringResponder(200, `{"payouts": [{"id":1, "date":"2022-02-03"}]}`))
```

Comment:
Is it worh having richer payouts and putting them in a json file and use load_fixture?

Reply:
Yes, I just followed some other tests to stay consistent, but I can improve the tests today.